This task is on Python Exception fro error handling using try, except, finally, raise and pass.
